.atomist/manifest.yml

```diff
@@ -1,6 +1,6 @@
 group: atomist-rugs
 artifact: common-editors
-version: "0.7.0"
-requires: "[0.8.0,1.0.0)"
+version: "0.8.0"
```

Comment:
0.7.0 is not released. The version in the manifest is the next version. I hate it too - and was what I was ranting about before.

https://github.com/atomist-rugs/common-editors/releases

We always bump the version post-release.